src/marlin_stubs/M865.cpp

```diff
@@ -23,12 +22,13 @@
  * - `P` - Nozzle preheat temperature
  * - `B` - Bed temperature
  * - `A` - Is abrasive
- * - `F` - Requries filtration
+ * - `F` - Requires filtration
  * - `N"<string>"` - New filament name
  *
  * Ad-hoc/custom filaments can the be referenced in other gcodes using adhoc_filament_gcode_prefix.
  * For example `M600 S"#0"` will load ad-hoc filament previously set with `M865 I0`.
- * The current filament settings of the specified tool (indexed from 0) are printed to the serial console
+ * The filament_type settings are printed to the serial console. For option I, the current filament_type
```

Comment:
Nitpick: Why do you keep referring to the `I` parameter here, like it's something special?